webapp/src/components/ProgramsPage.tsx

```diff
@@ -28,9 +28,13 @@ const ProgramsPage = () => {
   }
   if (isLoading) {
     return (
-      <Container fixed>
-        <div>Loading...</div>
-      </Container>
+      <Stack
```

Comment:
Looks great! I really liked that you maintained consistency and displayed the same image we used before.